osu.Game.Rulesets.Osu/Difficulty/Preprocessing/OsuDifficultyHitObject.cs

```diff
@@ -30,6 +30,11 @@ public class OsuDifficultyHitObject : DifficultyHitObject
         /// </summary>
         public readonly double StrainTime;
 
+        /// <summary>
+        /// Milliseconds elapsed since the start time of the previous <see cref="OsuDifficultyHitObject"/>, with a minimum of 25ms, adjusted with possible cheese.
```

Comment:
`...adjusted to compensate for possible difficulty reduction due doubletapping` or something